src/web/src/components/CourseSectionSeatsBadge.vue

```diff
@@ -0,0 +1,16 @@
+<template functional>
+  <b-badge v-if="props.seatsOpen <= 0" variant="danger">
+    {{ props.seatsOpen }} / {{ props.seatsTotal }} Full
```

Comment:
Lets maybe reduce duplication here with like a `.getStatus()` depending on seat count to get "Open" "Low" "Closed" and then a `.getClassFromStatus(statusEnum)` to map from status to class.

This would shrink down the template size by 70% and remove some duplication.

What you think?

Reply:
Yeah good points. I will try it out